SU-SWS/stanford_media
# D8CORE-1392 Added padding to aligned images

# READY FOR REVIEW

# Summary
- Added padding to aligned images.

# Need Review By (Date)
- 2/28

# Urgency
- low

# Steps to Test
1. checkout this branch
1. add an image to the wysiwyg and align it to the left or right.
1. add some text to wrap around the image.
1. save and verify there's padding. around the image.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
